tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -1308,6 +1308,20 @@ def test_deconstructible(self):
         self.assertEqual(args, (Point(0, 0), MultiPoint(Point(0, 0), Point(1, 1)), poly))
         self.assertEqual(kwargs, {})
 
+    def test_subclassing(self):
+        class ExtendedPolygon(Polygon):
```

Comment:
Possibly a docstring could be useful?  `"""GEOSGeometry may be subclassed ....""""`